build.sbt

```diff
@@ -64,28 +64,27 @@ lazy val wartRemoverSettings = {
 }
 
 lazy val compileDeps = Seq(
-  "uk.gov.hmrc"   %% "bootstrap-frontend-play-27"    % "5.11.0",
-  "uk.gov.hmrc"   %% "govuk-template"                % "5.69.0-play-27",
-  "uk.gov.hmrc"   %% "play-ui"                       % "9.6.0-play-27",
-  "uk.gov.hmrc"   %% "play-partials"                 % "8.2.0-play-27",
-  "uk.gov.hmrc"   %% "agent-kenshoo-monitoring"      % "4.8.0-play-27",
+  "uk.gov.hmrc"   %% "bootstrap-frontend-play-28"    % "5.9.0",
+  "uk.gov.hmrc"   %% "govuk-template"                % "5.70.0-play-28",
+  "uk.gov.hmrc"   %% "play-ui"                       % "9.7.0-play-28",
+  "uk.gov.hmrc"   %% "play-partials"                 % "8.2.0-play-28",
+  "uk.gov.hmrc"   %% "agent-kenshoo-monitoring"      % "4.8.0-play-28",
   "uk.gov.hmrc"   %% "agent-mtd-identifiers"         % "0.25.0-play-27",
-  "uk.gov.hmrc"   %% "mongo-caching"                 % "7.0.0-play-27",
-  "uk.gov.hmrc"   %% "play-conditional-form-mapping" % "1.9.0-play-27",
-  "uk.gov.hmrc"   %% "simple-reactivemongo"          % "8.0.0-play-27",
+  "uk.gov.hmrc"   %% "mongo-caching"                 % "7.0.0-play-28",
+  "uk.gov.hmrc"   %% "play-conditional-form-mapping" % "1.9.0-play-28",
+  "uk.gov.hmrc"   %% "simple-reactivemongo"          % "8.0.0-play-28",
   "uk.gov.hmrc"   %% "play-language"                 % "5.1.0-play-27",
```

Comment:
missed a play-28 here for play-language - but assuming agent-mtd-identifiers is fine since it's a micro-library and previous ones have been merged without issue

Reply:
Well spotted. Fixed!